sdk/internal/uuid/uuid.go

```diff
@@ -6,45 +6,28 @@
 package uuid
 
 import (
+	"crypto/rand"
 	"fmt"
-	"math/rand"
 	"strconv"
-	"time"
 )
 
 // The UUID reserved variants.
 const (
 	reservedRFC4122 byte = 0x40
 )
 
-func init() {
-	rand.Seed(time.Now().Unix())
-}
-
 // A UUID representation compliant with specification in RFC 4122 document.
 type UUID [16]byte
 
 // New returns a new uuid using RFC 4122 algorithm.
```

Comment:
I'm not sure about convention but if a function can panic I think it should definitely be documented as that's a very rare occurrence. Maybe it's just taken as a given that though.

CC: @JeffreyRichter , @jhendrixMSFT for idiom check

Reply:
[Docs](https://pkg.go.dev/math/rand#Read) mention that this is always a nil error, I think we can ignore the error for now. You will likely need a `//nolint` comment, the linter might complain about not checking a returned error.

Reply:
I think that doc is for math/rand (@RickWinter swapped to [crypto/rand](https://pkg.go.dev/crypto/rand))

Reply:
@richardpark-msft thanks for catching that the math/rand docs also make no mention of a long process, but I think we should be returning the error in this case.

Reply:
I spoke with Jeff on this change and its our conclusion that the error cases should never be hit given how we use the API.  If they are hit then we would want to stop (panic) so users can debug an unexpected and unrecoverable error

Reply:
This has been fixed to not panic.  I also fixed the `Parse()` function as it was swallowing errors.